blockchain/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Blockchain Template",
+    "description": "This template deploys a variety of blockchain services.",
+  "summary": "This template deploys a variety of blockchain services.",
```

Comment:
reword?
This Microsoft Azure Resource Manager template deploys a single VM and allows you to select from a number of blockchain software options to run on the VM.